test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -104,51 +108,50 @@ roundtripAndGolden = roundtripAndGoldenSpecsWithSettings settings
                               Arbitrary Instances
 -------------------------------------------------------------------------------}
 
-instance Arbitrary Amount where
-    shrink (Amount 0) = []
-    shrink _ = [Amount 0]
-    arbitrary = Amount . fromIntegral <$> (arbitrary @Word8)
+instance Arbitrary (MeasuredIn "lovelace" Natural) where
+    shrink (MeasuredIn 0) = []
+    shrink _ = [MeasuredIn 0]
+    arbitrary = MeasuredIn . fromIntegral <$> (arbitrary @Word8)
 
-instance Arbitrary Percentage where
-    arbitrary = arbitraryBoundedEnum
+instance Arbitrary (MeasuredIn "percent" Word) where
+    arbitrary = MeasuredIn <$> choose (0, 100)
```

Comment:
What is wrong with using `arbitraryBoundedEnum`? We have instances of `Bounded` and `Enum` defined, so why not use them here?

Reply:
Nothing, I forgot to move the `Enum` instance back in the code. I intended to move it to the test code because that's the only place where it's actually used from at the moment. `Enum` instances are quite a pain in the *** to test, so I'd rather not give extra power to the type in our code if we don't need it. It makes it easier to define the arbitrary instance though, I totally agree.